index.html

```diff
@@ -1563,9 +1562,8 @@ <h3 id="applying">
           </li>
           <li>If <a>Type</a>(<var>value</var>) is not "string":
             <ol>
-              <li>If <a>Type</a>(<var>value</var>) is not "undefined",
-              optionally <a>issue a developer warning</a> that the type is not
-              supported.
+              <li>If <a>Type</a>(<var>value</var>) is not "undefined", <a>issue
+              a developer warning</a> that the type is not supported.
```

Comment:
Actually this is not even in my code. This is just stuff I cleaned up as requested in an earlier code review to remove "optionally". So I'd prefer to not mess with this at all.